docs/sdk/wrapped-keys/generating-wrapped-key.md

```diff
@@ -0,0 +1,315 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Generating a New Key
+
+This guide covers the `generatePrivateKey` function from the Wrapped Keys SDK. For an overview of what a Wrapped Key is and what can be done with it, please go [here](./overview.md).
+
+Using the `generatePrivateKey` function, you can request a Lit node to generate a new private key within it's trusted execution environment (TEE). Once generated, the private key will be encrypted using Lit network's BLS key, and the resulting encryption metadata (`ciphertext` and `dataToEncryptHash`) will be returned and stored by Lit on your behalf.
```

Comment:
when u mention the TEE, i would link to the 'how it works' page here: https://developer.litprotocol.com/resources/how-it-works#1-lit-nodes